e2e/src/AccountManagement.spec.js

```diff
@@ -9,6 +9,9 @@ describe('Account', () => {
   })
 
   describe('Support', Support)
```

Comment:
not doing a describe.each here because not all tests need to run against both navs, see Support.js for tests running against both variants